gmail/endpoint.bal

```diff
@@ -46,40 +46,16 @@ public client class Client {
     #
     # + filter - Optional. MsgSearchFilter with optional query parameters to search messages.
     # + userId - The user's email address. The special value **me** can be used to indicate the authenticated user.
-    # + return - If successful, returns MessageListPage. Else returns error.
+    # + return - If successful, returns stream<Message,error>. Else returns error.
     @display {label: "List Messages"} 
     remote isolated function listMessages(@display {label: "Message Search Filter"} MsgSearchFilter? filter = (),
-                                           @display {label: "Email Address"} string? userId = ())
-                                           returns @tainted @display {label: "Message List"} MessageListPage|error {
+                                          @display {label: "Email Address"} string? userId = ()) returns @tainted 
+                                          @display {label: "Message List"} stream<Message,error>|error {
```

Comment:
@MaryamZi could you pls check if this should be `stream<Message,error?>` ?

Reply:
Yes, the stream's completion type should match that of the `next` method of the stream implementor. This doesn't fail atm due to a bug - https://github.com/ballerina-platform/ballerina-lang/issues/31464.

Reply:
@MaryamZi  So do we need to change to `stream<Message,error?>` now or do we need to wait until the fix comes ?

Reply:
I suggest using the correct type. If not, once the fix is available, this will have to be changed. Which will be a breaking change.

Reply:
Thanks. updated with the correct type.

Reply:
+1